restbridge/detail/RequestHandler.cpp

```diff
@@ -107,7 +106,7 @@ void RequestHandler::onStartupHeartbeatEvent_()
 
 void RequestHandler::onHeartbeatEvent_()
 {
-    LBVERB << "Heartbeat event received." << std::endl;
+    std::cout << "Heartbeat event received." << std::endl;
```

Comment:
LBVERB is something completely different than std::cout!! If you don't want to use Lunchbox, implement your own severity and filtering for messages. This is a no-go for me, sorry.